app/controllers/lunches/details/index.js

```diff
@@ -148,7 +148,7 @@ function showCamera() {
 	}
 	
 	// Send red sqare on Simulator
-	if (Ti.App.getDeployType() == 'development') {
+	if (utils.isEmulator()) {
```

Comment:
Basically, doing that,  if I test on my Device I'm still on development mode, and I always get the message **"You are currently running on the Simulator - without camera! We will generate a 1000x1000 red square now and try to send it to the server, do you really want to proceed?"** so this check is incorrect. If you want check for simulators like the message says the best way is the PR.